web-ui/styles/task-orchestration.css

```
@@ -1,2979 +1,475 @@
-#panel-orchestration .task-chat-panel {
-    display: flex;
-    flex-direction: column;
-    gap: 8px;
-    height: auto;
-    min-height: 0;
-    margin: 0 auto;
+#panel-orchestration.kilo-webview-mode {
+    --kilo-workbench-shell-offset: 204px;
+    --kilo-workbench-min-height: 560px;
+    --kilo-workbench-max-height: 980px;
+    min-height: min(var(--kilo-workbench-min-height), calc(100vh - var(--kilo-workbench-shell-offset)));
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)